CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to RKE #
+
+Thanks for taking the time to contribute to RKE!
+
+Of course, contributing code is more than welcome! To keep things simple, if you're fixing a small issue, you can simply submit a PR and we will pick it up. However, if you're planning to submit a bigger PR to implement a new feature or fix a relatively complex bug, please continue reading.
+
+### Opening PRs and organizing commits
+PRs should generally address only 1 issue at a time. If you need to fix two bugs, open two separate PRs. This will keep the scope of your pull requests smaller and allow them to be reviewed and merged more quickly.
+
+Each PR needs to be linked to a Github issue in [rancher project](https://github.com/rancher/rancher) which provides as much detail as possible on the commit at its motivation. If it is a bug fix, please describe how to reproduce it in the issue. Make the team or Project Manager aware of the Github issue so that they can set the correct milestone. Milestones are very important for release and QA tracking.
```

Comment:
1. Can we create a PR template for rancher/rke? It'd be a lot easier to direct contributors on what/what is required to add to a PR.
2. As I understand it, we are focusing much more on `what code has been modified/added` in the PR description to be able to catch regressions quicker, not just adding repro steps to the issue. I'd add a splurge about that to make that obvious to contributors. Our focus on regression prevention needs to extend to community PRs too.

Reply:
re: 1, yes, we could do that but in another PR. Could you create an issue about it?

re: 2. if you are referring to the line `If it is a bug fix, please describe how to reproduce it in the issue`. The reason is mainly that we should triage the issue to verify that it is indeed a general RKE1 issue. A lot of users have problems which end up being a misconfiguration in their hosts

Reply:
1. Yes
2. True, but that's an addition to the issue and this section is on 'Opening PRs and organizing commits' so I think it'd be helpful to add docs that will help the contributor focus on _what code/additions were added_ in the PR itself.

Reply:
I change opinion :). I think it is best to add the PR Template to this PR because it is related